app/main.py

```diff
@@ -1,2 +1,38 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.hash_table: list = [None] * 8
+
+    def _hash(self, key: Any) -> int:
+        return hash(key) % len(self.hash_table)
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        index = self._hash(key)
+
+        if self.hash_table[index] is None:
+            self.hash_table[index] = []
```

Comment:
Your dicitionary will only have max 8 elements inside? What if we want to store 18 elements?

Create `resize` logic to increase size of your dict